tesseract/tesseract_planning/tesseract_motion_planners/test/ompl_planner_tests.cpp

```diff
@@ -116,32 +116,170 @@ static void addBox(tesseract_environment::Environment& env)
   env.addLink(link_1, joint_1);
 }
 
-template <typename PlannerType>
-class OMPLTestFixture : public ::testing::Test
+void testOMPLPlanner(tesseract_motion_planners::OMPLPlannerFreespaceConfig::Ptr ompl_config,
+                     std::vector<std::string> joint_names)
 {
-public:
-  using ::testing::Test::Test;
-  tesseract_motion_planners::OMPLMotionPlanner<PlannerType> ompl_planner;
-};
-
-using Implementations = ::testing::Types<ompl::geometric::SBL,
-                                         ompl::geometric::PRM,
-                                         ompl::geometric::PRMstar,
-                                         ompl::geometric::LazyPRMstar,
-                                         ompl::geometric::EST,
-                                         ompl::geometric::LBKPIECE1,
-                                         ompl::geometric::BKPIECE1,
-                                         ompl::geometric::KPIECE1,
-                                         // ompl::geometric::RRT,
-                                         // ompl::geometric::RRTstar,
-                                         // ompl::geometric::SPARS,
-                                         // ompl::geometric::TRRT,
-                                         ompl::geometric::RRTConnect>;
-
-TYPED_TEST_CASE(OMPLTestFixture, Implementations);
-
-TYPED_TEST(OMPLTestFixture, OMPLFreespacePlannerUnit)
```

Comment:
You could probably retain the template test structure that currently exists and replace the implementations with the new configurator classes. This would cut down on the amount of repeat code you need to create here

Reply:
Switched.